src/SFA.DAS.ApprenticeCommitments/Data/Models/CourseDetails.cs

```diff
@@ -33,5 +33,12 @@ private CourseDetails()
         public DateTime PlannedStartDate { get; private set; }
 
         public int DurationInMonths => DifferenceInMonths(PlannedStartDate, PlannedEndDate);
+
+        public bool IsEquivalent(CourseDetails o)
```

Comment:
Could we use the built-in Equals method here?

Reply:
I don't think we should as it's not a struct. I did want to use a struct here, but EFCore didn't seem to like it and so I resorted to using a class. Also wasn't sure of the side effects of overloading equals on an EF class, so backed away for the moment. 

This was teh compromise

Reply:
I haven't used structs with EFCore before, but I have used overriden Equals methods without any issues (that I can remember!)

I'm happy to leave it as a custom method if it's easier, was just thinking that there's a standard method available.